README.md

```diff
@@ -47,6 +47,14 @@ docker:
     - "/path/to/another-image-name.tar"
 ```
 
+### Run a Command
+device-init can execute a list of commands on boot. e.g.:
+```yaml
+runcmd:
+  - "apt-get install package-name"
```

Comment:
The device-init.yaml is provided by end-user. So, he is responsible for providing a correct yaml that can be parsed